Week 4 Hangman/main.py

```diff
@@ -80,19 +80,29 @@ def generate(self) -> int:
 class User:
     def enterInput(self) -> str:
```

Comment:
Using in python instead of camel case we use lowercase where words are separated by underscores.
Same for variable also